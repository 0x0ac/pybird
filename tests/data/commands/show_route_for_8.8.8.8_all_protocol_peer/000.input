0001 BIRD 1.4.5 ready.
1007-8.8.8.0/24         via 206.41.110.37 on bond0.895 [chix_as33713_rs0 2016-09-16 from 206.41.110.66] (100) [AS15169i]
1008-   Type: BGP unicast univ
1012-   BGP.origin: IGP
        BGP.as_path: 15169
        BGP.next_hop: 206.41.110.37
        BGP.med: 0
        BGP.local_pref: 200
        BGP.community: (63311,21201)
0000
