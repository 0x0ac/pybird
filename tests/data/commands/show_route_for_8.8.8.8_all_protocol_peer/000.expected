[{"origin": "IGP", "med": "0", "as_path": "15169", "community": "63311:21201", "prefix": "8.8.8.0/24", "next_hop": "206.41.110.37", "peer": "206.41.110.37", "local_pref": "200"}]
